docs_src/OVMS/supportingDifferentModel.md

```diff
@@ -13,4 +17,14 @@ OvmsClient:
   # PipelineInputArgs: "--model_name bit_64" # space delimited like we run the script in command and take those input arguments
   # PipelineInputArgs: "--model_name yolov5s" # space delimited like we run the script in command and take those input arguments
 
-```
\ No newline at end of file
+```
+
+## Download Models
+You can download models by editing `download_models/models.lst` file, you can add new models(from https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/object_detection_demo/python/models.lst) to it or uncomment from existing list in this file, saved the file once editing is done. Then you can download the list using following steps:
+
+1. `cd download_models`
+2. `make build`
+3. `make run`
+
+after above steps, the downloaded models can be found in `configs/opencv-ovms/models/2022` directory.
+**_Note:_** model files in `configs/opencv-ovms/models/2022` directory will be replaced with new downloads if previously existed.
```

Comment:
how do i use this to download instance -segmentation and bit model? they are not covered in the list

Reply:
unfortunately open model zoo omz_downloader doesn't recognize these models, but luckily, we already have them in our code base. if our future OVMS models will be aligned with open model zoo's, then maybe we should look forward to open model zoo's models and just leave what we had.